src/dinit.cc

```diff
@@ -453,6 +471,68 @@ static int process_commandline_arg(char **argv, int argc, int &i, options &opts)
     return 0;
 }
 
+#if SUPPORT_SELINUX
+static bool selinux_transition(const char *exe) {
+    using std::cerr;
+    using std::endl;
+
```

Comment:
There should be an explanation (via comment) of why output is via `cerr` instead of the log. (Basically: because if we output anything we are returning failure, and in that case expect that dinit will terminate before the log is initialised and flushed).

Reply:
> which indicates dinit should terminate before the log is initalised and flushed.

I am being a little bit picky here, but this could be clearer. I think what you mean is that it _indicates dinit should terminate, which may happen before the log is initialised and flushed_.

The way it is written it sounds like returning false specifically indicates that the log should not be initialised and flushed before dinit terminates. But actually, whether the log was initialised and flushed before terminating is not important.